internal/featuresupport/support_levels_list.go

```diff
@@ -99,6 +99,11 @@ var SupportLevelsList = models.FeatureSupportLevels{
 				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelDevPreview,
 			},
 			// Unsupported features
+			{
+				// ODF is marked as unsupported only for v2.1 and because we don't have the ODF code changes in this version
+				FeatureID:    usageNameToID("odf"),
```

Comment:
hmm, wait
I think we still call it OCS in this version (remember it's before ODF changes)

Reply:
+1 Osher

Reply:
After discussing with @batzionb about it we decided to leave it as is (ODF) due to that currently there are already places on UI that it called ODF and not OCS

Reply:
so it seems like it should be:
```suggestion
				FeatureID:    usageNameToID("odf-operator"),
```
following how you've wrote it in the swagger